hrui/js/src/hrui.js

```diff
@@ -1,11 +1,115 @@
+/* global Ps */
+
 // Copyright CiviCRM LLC 2013. See http://civicrm.org/licensing
 (function ($, _) {
+  $('.CRM_HRRecruitment_Form_Application').addClass('crm-form-block');
```

Comment:
This file is a mess right now, with no proper structure to the code. Can we clean this up a bit.
```js
(function init () {
  $(document).on('crmLoad', crmOnLoad);
  $(document).ready(onReady);
})();

//Keep all the private functions below
// Also can we break crmOnLoad into smaller chunks